packages/extension/src/view/devtools/components/cookies/cookieTopBar/index.tsx

```diff
@@ -51,10 +64,82 @@ const CookieSearch = ({
     })
   );
 
-  const { getCookiesSetByJavascript } = useCookieStore(({ actions }) => ({
+  const cookieDeletedRef = useRef(false);
```

Comment:
```suggestion
  const isCookieDeleted = useRef(false);
```

Reply:
fixed in [9de5a3d](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/278/commits/9de5a3df8410e59c636e0d8210f8ff5dea009c17)